src/main/antlr/STParser.g4

```diff
@@ -36,7 +36,7 @@ parser grammar STParser;
 
 options {
 	language=Java;
-	tokenVocab=STLexer;
+//	tokenVocab= STLexer;
```

Comment:
I don't think this should be commented out

Reply:
I believe you should uncomment the line and add this option to the `generateGrammarSources` target:

```
-lib src/main/antlr
```

Reply:
agree, I've add this to the `build.gradle`